docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -2,4 +2,182 @@
 Developer Setup
 ***************
 
-TBD
+Creating a development shell
+============================
+
+To create an interactive development ``bash`` shell:
+
+This recipe uses the ``aarch64`` (64-bit ARM) Miniforge for Linux, and installs into ``$HOME/conda``. Adjust as necessary for your target system.
+
+1. Download, install, and activate the latest `Miniforge3`_ for your system. If an existing conda (Miniforge, Miniconda, Anaconda, etc.) installation is available and writable, you may activate that and skip this step and continue on to the next.
+
+   .. code:: sh
+   
+     wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh
+     bash Miniforge3-Linux-aarch64.sh -bfp ~/conda
+     rm Miniforge3-Linux-aarch64.sh
+     source ~/conda/etc/profile.d/conda.sh
+     conda activate
+
+2. Install the `condev`_ package into the base environment.
+
+   .. code:: sh
+
+     conda install -y -c maddenp condev
+
+3. In a clone of the `workflow-tools repository`_, create the development shell.
+
+
+   .. code:: sh
+
+     cd /to/your/workflow-tools/clone
+     make devshell
+
+
```

Comment:
The previous code block was broken down to include the code relevant to the step it is now encompassed under. 

I attempted to update the line for skipping the first step in the process to "installation is available and writable, you may activate that and skip this step and continue on to the next." Let me know if that looks more readable and less confusing hopefully.